app/src/main/java/org/oppia/android/app/help/faq/faqsingle/FAQSingleActivity.kt

```diff
@@ -16,8 +16,12 @@ class FAQSingleActivity : InjectableAppCompatActivity() {
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
     (activityComponent as ActivityComponentImpl).inject(this)
-    val question = checkNotNull(intent.getStringExtra(FAQ_SINGLE_ACTIVITY_QUESTION))
-    val answer = checkNotNull(intent.getStringExtra(FAQ_SINGLE_ACTIVITY_ANSWER))
+    val question = checkNotNull(intent.getStringExtra(FAQ_SINGLE_ACTIVITY_QUESTION)) {
+      "Expected FAQ_SINGLE_ACTIVITY_QUESTION to be in intent extras."
+    }
+    val answer = checkNotNull(intent.getStringExtra(FAQ_SINGLE_ACTIVITY_ANSWER)) {
+      "Expected FAQ_SINGLE_ACTIVITY_ANSWER to be in intent extras."
```

Comment:
```suggestion
      "Expected $FAQ_SINGLE_ACTIVITY_ANSWER to be in intent extras."
```

Ditto for the other (since this is the actual name of the extra), and in other files in this PR.

Reply:
Fixed